app/views/users/show.html.erb

```diff
@@ -1,71 +1,68 @@
-<div class="page page-dashboard">
+<div class="page page-dashboard page-dashboard-light">
   <div class="wrapper wrapper--light">
     <div class="container">
       <div class="pagehead">
         <div class="row">
-          <div class="col-sm-6">
+          <div class="col-xs-12 col-flex">
             <ol class="breadcrumb">
-              <li class="active">My Profile</li>
+              <li class="active">Dashboard</li>
             </ol>
           </div>
-          <div class="col-sm-6">
-            <div class="btn-toolbar pull-right">
-              <div class="btn-group">
-                <%= link_to 'Edit Profile', edit_user_registration_path(@user), class: 'btn btn-default' %>
-              </div>
-              <div class="btn-group pull-right">
-                <%= link_to 'Dashboard', dashboard_user_path(@user), class: 'btn btn-brand' %>
-              </div>
+        </div>
+      </div>
+    </div>
+  </div>
+  <div class="wrapper wrapper--light">
+    <div class="container">
+      <div class="pagehead">
+        <div class="row">
+          <div class="col-xs-12">
+            <label>Current Decks</label>
+            <div class="grid">
+              <% @decks.take(10).each_with_index do |deck, index| %>
+                <%= render 'dashboard_current_deck', deck: deck, index: index %>
+              <% end %>
```

Comment:
I'm using index to mark the first card as `block-last-active`. This is all placeholder data and will be removed in the future